Mathlib/SetTheory/Cardinal/Ordinal.lean

```diff
@@ -1071,7 +1071,6 @@ theorem mk_multiset_of_nonempty (α : Type u) [Nonempty α] : #(Multiset α) = m
 theorem mk_multiset_of_infinite (α : Type u) [Infinite α] : #(Multiset α) = #α := by simp
 #align cardinal.mk_multiset_of_infinite Cardinal.mk_multiset_of_infinite
 
-@[simp]
```

Comment:
I guess the linter now complains because this follows by uniqueness?

Reply:
Yeah, as shown by [this CI run](https://github.com/leanprover-community/mathlib4/actions/runs/6916053475/job/18815681354).